hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/blockmanagement/BlockManager.java

```diff
@@ -5190,7 +5191,7 @@ public void run() {
         processQueue();
       } catch (Throwable t) {
         ExitUtil.terminate(1,
-            getName() + " encountered fatal exception: " + t);
+            getName() + " encountered fatal exception: " + ExceptionUtils.getStackTrace(t));
```

Comment:
looking at other usage in the Hadoop codebase, it seems to me a better way to print stack trace is to use the logger.
https://github.com/apache/hadoop/blob/d0dcfc405c624f73ed1af9527bbf456a10337a6d/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/hadoop-yarn-services/hadoop-yarn-services-core/src/main/java/org/apache/hadoop/yarn/service/ServiceMaster.java#L353

Reply:
Thanks @jojochuang  for the quick review & feedback.
I have addressed the changes and pushed the changes. Please review.